src/renderer/desktop-capturer.ts

```diff
@@ -139,16 +139,9 @@ export const getSource = async (
     }
   }
 
-  const updatedSources = sources
-    .filter((source) => source.name !== NOTIFICATION_WINDOW_TITLE)
-    .map((source) => {
-      return {
-        ...source,
-        ...{
-          thumbnail: source.thumbnail.toDataURL(),
```

Comment:
Why can't `thumbnail` be the interface implementation instead of overriding it? I think it's just unnecessary.